src/containers/user-profile/profile-info/ProfileContainerDesktop.jsx

```diff
@@ -18,6 +18,7 @@ const ProfileContainerDesktop = ({
   doneItems,
   chipItems
 }) => {
+  console.log(userData.professionalSummary)
```

Comment:
Do we need this console.log ?

Reply:
I did not notice it, thanks for your thorough review. Removed redundant `console.log`